Namco/pacman.py

```diff
@@ -29,6 +29,24 @@ def step(self):
     def render(self):
         pass
 
+    def movement(self,action,x_pos,y_pos):
+        # going up
+        if action == 'w':
+            pos_x -= 1
```

Comment:
Amikor beállítjuk az egyik irányú sebességet, a másikat 0-zni kellene minden esetben.

Reply:
Ha sebességként kezeljük akkor igen, én elsőre azt gondoltam, hogy kordinátaként vesszük. Ugye azért is ez jutott eszembe, mert ahogy olvastam az automatikus léptetésnél azt magyaráztátok, hogy ha x másodpercig nincs más input akkor megint lépjen egyet az előző action szerint. De valószinűleg egyszerűbb ha csak sebesség.

Reply:
Jaa, amúgy így is jó igazából. Fölösleges egyelőre a sebesség, elég ha a pozíció állítódik. Aztan max ha később valamiért mégsem lesz jó, akkor majd változtatjuk. Viszont ha pozíciót használunk csak, akkor itt kéne lekezelni, hogy ne tudjon kimenni a pályáról.